react-stopwatch/my-app/src/App.tsx

```diff
@@ -0,0 +1,32 @@
+import { useState } from 'react';
+import './App.css';
+import { FaPlay, FaPause } from 'react-icons/fa';
+
+function App() {
+  const [elapsedSeconds, setElapsedSeconds] = useState(0);
+  const [intervalId, setIntervalId] = useState<NodeJS.Timeout>();
+  function handleClick() {
+    if (!intervalId) {
+      const intervalId = setInterval(
+        () => setElapsedSeconds((prev) => prev + 1),
+        1000
+      );
+      setIntervalId(intervalId);
+    } else {
+      clearInterval(intervalId);
+      setIntervalId(undefined);
+    }
+  }
+  return (
+    <div className="container">
+      <div className="stopwatch" onClick={() => setElapsedSeconds(0)}>
```

Comment:
The logic needs to be a little more sophisticated than just setting seconds to 0, since you only want to do that if the watch is stopped. If it is running, you want to ignore the click.